modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/entries/display/context/ObjectEntryDisplayContextImpl.java

```diff
@@ -1495,6 +1495,17 @@ private ObjectEntry _getObjectEntry() throws PortalException {
 				ObjectWebKeys.OBJECT_ENTRY_EXTERNAL_REFERENCE_CODE);
 		}
 
+		HttpServletRequest httpServletRequest =
+			_objectRequestHelper.getRequest();
+
+		Long groupId = GetterUtil.getLong(
```

Comment:
Can you use the primitive type long? the method `GetterUtil.getLong(` returns a `long`.